mongo/document/document.go

```diff
@@ -65,7 +65,7 @@ func Create(ctx context.Context, collectionName string, doc document) error {
 
 	c := mongo.Session(log).Clone().DB("").C(collectionName)
 	defer c.Database.Session.Close()
-	log.WithField(collectionName, doc).Debugf("save '%v'", collectionName)
+	log.WithField("collection", collectionName).Debugf("save '%v'", collectionName)
```

Comment:
Could you add the Document ID ? This is not sensitive information and can be very useful for debugging.